app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -53,7 +53,7 @@ def cluster(cluster)
     end
 
     def clusters
-      dc.clusters.map(&:full_path).sort
+      dc.clusters.map(&:full_path).sort rescue []
```

Comment:
Maybe to explicitly check for nil there logging the info message rather than "catch all and forget".